concepts/while-loops/about.md

```diff
@@ -0,0 +1,83 @@
+# About
+
+## General Syntax
+
+With a while loop you can execute code repeatably as long as a certain condition is fulfilled.
+
+It is written with the `while` keyword followed by a condition wrapped in round brackets and a code block that contains the _body_ of the loop wrapped in curly brackets.
+
+```javascript
+while (condition) {
+  // code that is executed repeatedly as long as the condition is true
+}
+```
+
+JavaScript also has a do-while loop.
+Here the condition is evaluated after the loop body was executed.
+This is useful when the condition depends on the outcome of the code in the body.
```

Comment:
```suggestion
This is useful when the condition depends on the outcome or evaluation of the body.
```

Reply:
good point, changed your suggestion a bit before putting it in